roles/activemq-packages-install/tasks/main.yml

```diff
@@ -22,13 +22,19 @@
     group: root
     mode: 0644
   become: yes
-  when: ansible_distribution == "CentOS" or ansible_distribution == "Red Hat Enterprise Linux"
+  when: ansible_distribution == "CentOS" or ansible_distribution == "Red Hat Enterprise Linux" or ansible_distribution == "Amazon"
 
+- name: Enable ActiveMQ to start at bootup
```

Comment:
Isn't this part of the AMQ package by default?

Reply:
I believe it's disabled by default, It'll do no harm.

```
activemq.service - Messaging and Integration Patterns provider
   Loaded: loaded (/usr/lib/systemd/system/activemq.service; enabled; vendor preset: disabled)
```

Reply:
Again, I've moved it from the handler as it is persistent state